tck/src/test/java/org/eclipse/microprofile/jwt/tck/config/PublicKeyEndpoint.java

```diff
@@ -559,6 +568,55 @@ else if(issuer.isPresent()) {
             .build();
         return result;
     }
+    
+    /**
+     * Check a token with an aud claim matches one of the mp.jwt.verify.audiences values
+     * @return result of validation test
+     */
+    @GET
+    @Path("/verifyAudIsOk")
+    @Produces(MediaType.APPLICATION_JSON)
+    @RolesAllowed("Tester")
+    public JsonObject verifyAudIsOk() {
+        boolean pass = false;
+        String msg;
+
+        if(!aud.getValue().isPresent()) {
+            // The iss claim should be provided for this endpoint
+            msg = String.format("MP-JWT missing aud claim, or injection of claim failed");
```

Comment:
String.format without placeholder. Can be simple String.

Reply:
fixed, thanks